core/app/models/spree/store.rb

```diff
@@ -17,6 +17,8 @@ class Store < Spree::Base
 
     has_many :orders, class_name: "Spree::Order"
 
+    belongs_to :default_user_group, class_name: 'Spree::UserGroup', optional: true
```

Comment:
Please align the naming of default user group with the documentation in the issue